index.rmd

```diff
@@ -1,27 +1,29 @@
 --- 
 title: "R packages"
-author: "Hadley Wickham"
-date: "`r Sys.Date()`"
+author: ["Hadley Wickham", "Jenny
```

Comment:
missing paired `]` here ?

Reply:
Gah, don't know how I managed to delete that